kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/rest/ProcessServiceRestOnlyIntegrationTest.java

```diff
@@ -94,4 +98,72 @@ public void testAbortAlreadyAbortedProcess() {
             }
         }
     }
+
+    @Test
+    public void testGetProcessInstances() {
+        Map<String, Object> valuesMap = new HashMap<String, Object>();
+        valuesMap.put(RestURI.CONTAINER_ID, CONTAINER_ID);
+        valuesMap.put(RestURI.PROCESS_ID, PROCESS_ID_USERTASK);
+
+        Map<String, Object> valuesGetMap = new HashMap<String, Object>();
+        valuesGetMap.put(RestURI.CONTAINER_ID, CONTAINER_ID);
+
+        Response response = null;
+        try {
+            // start process instance
+            WebTarget clientRequest = newRequest(build(TestConfig.getKieServerHttpUrl(), PROCESS_URI + "/" + START_PROCESS_POST_URI, valuesMap));
+            logger.debug("[POST] " + clientRequest.getUri());
+            response = clientRequest.request(getMediaType()).post(createEntity(""));
+            Assert.assertEquals(Response.Status.CREATED.getStatusCode(), response.getStatus());
+
+            Long resultOne = response.readEntity(JaxbLong.class).unwrap();
+            assertNotNull(resultOne);
+
+            // get processes instances
+            WebTarget clientGetRequest = newRequest(build(TestConfig.getKieServerHttpUrl(), PROCESS_URI + "/" + PROCESS_INSTANCES_BY_CONTAINER_GET_URI, valuesGetMap));
+            logger.debug("[GET]" + clientGetRequest.getUri());
+
+            response = clientGetRequest.request(getMediaType()).get();
+            Assert.assertEquals(Response.Status.OK.getStatusCode(), response.getStatus());
+
+            List<ProcessInstance> resultList = response.readEntity(ProcessInstanceList.class).getItems();
```

Comment:
test fails on jenkins here with "javax.ws.rs.ProcessingException: org.codehaus.jackson.map.JsonMappingException: Can not deserialize instance of java.util.Date out of START_OBJECT token"